Create ui-c8y-1023-42-2-remove-additional-space-before-ending-dot.md (#4263)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-42-2-remove-additional-space-before-ending-dot.md

---
date: ""

> title: Remove additional space before ending dot.

Reviewer: ```suggestion
title: Remove extra space before ending punctuation in empty state components
```

> title: Remove extra space before ending punctuation in empty state components

Reviewer 2: ```suggestion
title: Removed extra space before end punctuation marks in empty state components
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65698
version: 1023.42.2
---

> There was an extra empty space between link and ending dot- for example in Linear gauge widget config view preview (empty state). It was caused by invalid formatting of html templates. It is fixed for this and similiar cases. It also fixes future formatting.

Reviewer: ```suggestion
When viewing empty state components, e.g. widget configuration previews, such as the Linear gauge widget's empty state, an extra space was appearing between text and the ending punctuation mark (for example, "text ."). This was caused by improper formatting in the HTML templates used to render these previews. The underlying template formatting has been corrected to eliminate these extra spaces, which also prevents similar spacing issues from occurring in future widget previews and other UI elements that use the same templates.
```

> When viewing empty state components, e.g. widget configuration previews, such as the Linear gauge widget's empty state, an extra space was appearing between text and the ending punctuation mark (for example, "text ."). This was caused by improper formatting in the HTML templates used to render these previews. The underlying template formatting has been corrected to eliminate these extra spaces, which also prevents similar spacing issues from occurring in future widget previews and other UI elements that use the same templates.

Reviewer 2: ```suggestion
When viewing empty state components, such as widget configuration previews, for example, the "Linear gauge" widget's empty state, an extra space appeared between the text and the end punctuation mark (for example, "text ."). This was caused by improper formatting in the HTML templates used to render these previews. The underlying template formatting has been corrected to eliminate these extra spaces, which also prevents similar spacing issues from occurring in future widget previews and other UI elements that use the same templates.
```

> When viewing empty state components, such as widget configuration previews, for example, the "Linear gauge" widget's empty state, an extra space appeared between the text and the end punctuation mark (for example, "text ."). This was caused by improper formatting in the HTML templates used to render these previews. The underlying template formatting has been corrected to eliminate these extra spaces, which also prevents similar spacing issues from occurring in future widget previews and other UI elements that use the same templates.

Reviewer 2: ```suggestion
When viewing empty state components, such as widget configuration previews (for example, the "Linear gauge" widget's empty state) an extra space appeared between the text and the end punctuation mark (for example, "text ."). This was caused by improper formatting in the HTML templates used to render these previews. The underlying template formatting has been corrected to eliminate these extra spaces, which also prevents similar spacing issues from occurring in future widget previews and other UI elements that use the same templates.
```
